prow/config.yaml

```diff
@@ -3808,6 +3861,59 @@ presubmits:
     max_concurrency: 0
     name: pull-security-kubernetes-cross
```

Comment:
This job looks identical to the one that follows besides the branch sharding?

Reply:
the `run_if_changed` regex is wider for older banches (it's the existing regex which matches some go files, we test this with typecheck on master / 1.10+ now.